recipes/rosco/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "rosco" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nrel/rosco/archive/v{{ version }}.tar.gz
+  sha256: fc4813439be6cbf275cee1fa884e521490559ca87d2f63498ae4fab2dfa9bc48
+
+build:
+  number: 0
+  skip: False  # [win]
```

Comment:
```suggestion
```

Reply:
do you want to build it just for ``windows``?

Reply:
removed

Reply:
Would like it to build for linux, osx, and windows